### PR TITLE
fix(user rights): replace table with oui-datagrid

### PR DESCRIPTION
**Pull request by frenautvh**

### CODE REVIEW

File: client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html [hunk 1]
```diff
@@ -23,91 +23,34 @@ <h2 data-ng-bind-html="'dedicatedCloud_USER_rights_management_by_datacenter' | t
...
+                </span>
+            </oui-column>
+            <oui-column data-title=":: 'dedicatedCloud_USER_VM_NETWORK_ROLE' | translate">
+                <span data-translate="{{ 'dedicatedCloud_USER_right_' + $row.vmNetworkRole }}"></span>
+            </oui-column>
+            <oui-column data-title=":: 'dedicatedCloud_USER_NETWORK_ROLE' | translate">
+                <span data-translate="{{ 'dedicatedCloud_USER_right_' + $row.networkRole }}"></span>
+            </oui-column>
+            <oui-action-menu align="end" compact>
+                <oui-action-menu-item text="{{:: 'dedicatedCloud_USER_right_modify' | translate }}"
```
Comment (cbourgois) [inline client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html]: Should use `data-text` / `data-on-click`

File: client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html [hunk 2]
```diff
@@ -23,91 +23,34 @@ <h2 data-ng-bind-html="'dedicatedCloud_USER_rights_management_by_datacenter' | t
...
+                      data-ng-if="!$row.canAddResource">
+                </span>
+            </oui-column>
+            <oui-column data-title=":: 'dedicatedCloud_USER_VM_NETWORK_ROLE' | translate">
+                <span data-translate="{{ 'dedicatedCloud_USER_right_' + $row.vmNetworkRole }}"></span>
+            </oui-column>
+            <oui-column data-title=":: 'dedicatedCloud_USER_NETWORK_ROLE' | translate">
+                <span data-translate="{{ 'dedicatedCloud_USER_right_' + $row.networkRole }}"></span>
+            </oui-column>
+            <oui-action-menu align="end" compact>
```
Comment (cbourgois) [inline client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html]: Sould use `data-align` / `data-compact`

File: client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html (hunk 1 above)
Comment (FredericEspiau) [reply client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html]: Why isn't this detected by a linter :/

File: client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html [hunk 3]
```diff
@@ -23,91 +23,34 @@ <h2 data-ng-bind-html="'dedicatedCloud_USER_rights_management_by_datacenter' | t
...
-                </table>
-                <div data-pagination-server-side="rightTable"
-                     data-pagination-server-side-function="$ctrl.loadUserRight"
-                     data-pagination-server-side-paginated-stuff="$ctrl.rights"
-                     data-pagination-server-side-table-loading="$ctrl.loading.rights">
-                </div>
-            </div>
-        </div>
-    </div>
+        <oui-datagrid rows-loader="$ctrl.loadUserRights($config)">
```
Comment (cbourgois) [inline client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html]: Same here, should use `data-rows-loader`

**Pull request merged**